src/Symfony/Bundle/DependencyInjection/Configuration.php

```diff
@@ -426,6 +426,42 @@ private function addHttpCacheSection(ArrayNodeDefinition $rootNode): void
                                     ->canBeDisabled()
                                     ->info('Enable support for Cache-Tags invalidation.')
                                 ->end()
+                                ->arrayNode('purger')
+                                    ->addDefaultsIfNotSet()
+                                    ->info('Define the purger configuration.')
+                                    ->children()
+                                        ->scalarNode('dsn')
+                                            ->defaultValue('')
+                                            ->info('Purger dsn')
+                                        ->end()
+                                        ->scalarNode('path')
+                                            ->defaultValue('')
+                                            ->info('Purger API path')
+                                        ->end()
+                                        ->scalarNode('token')
+                                            ->defaultValue('')
+                                            ->info('JWT token to use if the provider API if secured')
```

Comment:
That's weird, shouldn't this be under a `souin` section as it's relevant only in this context. If it's just a bearer token, then the should be renamed to make it explicit that there is no hard dependency on Souin.